docs/visualization.rst

```diff
@@ -97,10 +101,13 @@ Setting the x and y ranges of the plot can be done simply by indexing the `~ndcu
   >>> plt.cla()
   >>> plt.close()
 
-Note how no arguments are required, the necessary information to generate the plot is derived from the data and metadata in the `~ndcube.NDCube`.
-However optional keywords enable customization of the visualization.
+Note that sometimes axis tickmarks are missing.
+This is a caused by a bug in `~astropy.visualization.wcsaes.WCSAxes` whereby the ticks and labels are omitted if the plot extends beyond the valid range of the WCS projection.
```

Comment:
This isn't a bug, as I discovered a while ago. It's in general not possible to display the ticks/labels if the spine of an axis is outside the range of the WCS projection.

Reply:
I think there are a few cases where wcsaxes could do better, but yea there are many where it's not a bug.

Reply:
Do you have any cases in mind? I'd be interested in taking a look and trying to improve things if there are things to be improved.

Reply:
```suggestion
This is a caused by a behavior in `~astropy.visualization.wcsaes.WCSAxes` whereby the ticks and labels are omitted if the plot extends beyond the valid range of the WCS projection.
```

Reply:
> Do you have any cases in mind? I'd be interested in taking a look and trying to improve things if there are things to be improved.

Well the 1D case (without thinking about it too much) seems particularly silly, how do you reach the edge of a projection in 1D??

Also I feel like for some of the whole sky projections where the coordinates are *so close* to the edge of the projection WCSAxes could be smarter, but I think this was the case you looked at before?